decred/tests/unit/dcr/wire/test_msgpong.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details.
+
+Based on dcrd MsgPing.
+"""
+
+import random
+
+from decred.dcr.wire import wire
+from decred.dcr.wire.msgpong import MsgPong
+from decred.util.encode import ByteArray
+
+
+def test_PongLatest():
```

Comment:
The name of the test is different than ping.

Not that that's too important. But just test_Pong is ok imo. Although, I hadn't been using camel case in test names. Again, feel free to ignore this.